Add project security policy SECURITY.md

otkd opened the pull request:
Fairly boilerplate, but worth having imo, @szh wasn't sure what your owasp email was

# You're A Rockstar

Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series.

> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double-check the file for other mistakes in order to fix all the issues in the current cheat sheet.

Please make sure that for your contribution:

- [ ] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [ ] All the markdown files do not raise any validation policy violation, see the [policy](https://github.com/OWASP/CheatSheetSeries/actions?query=workflow%3A%22Markdown+Link+Check%22).
- [ ] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries/blob/master/CONTRIBUTING.md#markdown).
- [ ] All your assets are stored in the **assets** folder.
- [ ] All the images used are in the **PNG** format.
- [ ] Any references to websites have been formatted as `[TEXT](URL)`
- [ ] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
- [ ] The CI build of your PR pass, see the build status [here](https://github.com/OWASP/CheatSheetSeries/actions).

If your PR is related to an issue, please finish your PR text with the following line:

This PR fixes issue #`<REPLACE WITH ISSUE NUMBER>`.

Thank you again for your contribution :smiley:

kwwall reviewed (approved):
LGTM, but you may want to explicitly note to not report vulnerabilities in dependencies, since CheatSheatSeries is already using Dependabot. We still get a lot of that on ESAPI even thought I use both Dependabot and Snyk to check, and users end up reporting those as regular issues (or on rare occasions, they will email them to me as vulnerabilities). I think that they generally do that because they are using some COTS SCA tool that reports the vulnerability a day or 2 before Synk or Dependabot does. Anyhow, this looks good. That's the only part that you may want to clarify, but I don't think it's as a big of a deal as it is for ESAPI.

szh commented:
This is great. I agree with @kwwall but not a blocker.
My OWASP email is shlomo.heigh

otkd commented:
@kwwall good point, added conditional where we may want to know about them sooner to address with more urgency

@szh thanks, fixed and added your email

szh reviewed (approved):
Great work, thank you!
LGTM
